patches/add_screenshots.patch

```diff
@@ -0,0 +1,35 @@
+From d7caf4367ea27bf36d3f6617e6ee3217e351214b Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Daniel=20Novomesk=C3=BD?= <dnovomesky@gmail.com>
+Date: Tue, 4 Mar 2025 19:17:53 +0100
+Subject: [PATCH] add screenshots to org.nomacs.ImageLounge.appdata.xml.in
+
+---
+ .../org.nomacs.ImageLounge.appdata.xml.in         | 15 ++++++++++-----
+ 1 file changed, 10 insertions(+), 5 deletions(-)
+
+diff --git a/ImageLounge/xgd-data/org.nomacs.ImageLounge.appdata.xml.in b/ImageLounge/xgd-data/org.nomacs.ImageLounge.appdata.xml.in
+index 3990415dc..eb1dd7bfb 100644
+--- a/ImageLounge/xgd-data/org.nomacs.ImageLounge.appdata.xml.in
++++ b/ImageLounge/xgd-data/org.nomacs.ImageLounge.appdata.xml.in
+@@ -32,11 +32,16 @@
+         <content_attribute id="language-profanity">mild</content_attribute>
+     </content_rating>
+ 
+-    <!-- TODO: new screenshots -->
+-    <!-- <screenshots> -->
+-    <!--     <screenshot type="default">https://nomacs.org/wp-content/uploads/2020/03/img-2020-03-18-11.49.26-1.png</screenshot> -->
+-    <!--     <screenshot type="default">https://nomacs.org/wp-content/uploads/2020/03/img-2020-03-18-11.59.10.png</screenshot> -->
+-    <!-- </screenshots> -->
++    <screenshots>
++        <screenshot type="default">
++            <image>https://github.com/nomacs/nomacs/raw/refs/heads/master/ImageLounge/screenshots/nomacs_3_21-thumbnails.png</image>
```

Comment:
this should NEVER reference the tip of a branch as it is not immutable.

Reply:
How the URLs should look like?

This is OK?

https://github.com/nomacs/nomacs/raw/fbb4e3ef48f5264b15ebb16521a105d4518e6000/ImageLounge/screenshots/nomacs_3_21-thumbnails.png

https://github.com/nomacs/nomacs/raw/fbb4e3ef48f5264b15ebb16521a105d4518e6000/ImageLounge/screenshots/nomacs_3_21-raw_metadata.png

Reply:
Yes